support-frontend/app/cookies/DigitalSubscriptionCookies.scala

```diff
@@ -0,0 +1,36 @@
+package cookies
+
+import config.Configuration.GuardianDomain
+import org.joda.time.DateTime
+import play.api.mvc.Cookie
+
+object DigitalSubscriptionCookies {
+  def create(domain: GuardianDomain): List[Cookie] = {
+    def cookie(name: String, value:String) = Cookie(
+      name = name,
+      value = value,
+      secure = true,
+      httpOnly = false,
+      domain = Some(domain.value)
+    )
+    List(
+      cookie(
+        name = "gu_user_features_expiry",
```

Comment:
would be nice to have a link to the user feaatures JS file in dotcom, so that people know it can be set in more than one place